uint8 local_addr
uint16 full_index
int16[3] initial_position
int16[4] initial_orientation
uint16[6] initial_sigmas